cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -78,6 +79,40 @@ def issue_jwt_token_with_jwk(token_body, file_name, algorithm = Algorithms::RS25
     )
   end
 
+  def issue_jwt_token_with_x5c(token_body, algorithm = Algorithms::RS256)
+    cert = self_signed_certificate(rsa_keys[@default_file_name])
+
+    @jwt_token = JWT.encode(
+      token_body,
+      rsa_keys[@default_file_name],
+      algorithm,
+      {
+        kid:  base64_x5t_from_certificate(cert),
```

Comment:
let's remove kid. it's unnecessary there

Reply:
if i remove that `kid` ill get `No key id (kid) found from token headers` error, which kind of misses the point of the tests

Reply:
we've discussed it. we're trying to create a valid token that's using `x5c` and `x5t` claims. So the assumption that it's not using `kid` if so.